authservice/auth_service.go

```diff
@@ -229,6 +255,11 @@ func NewAuthService(serviceID string, serviceHost string, serviceLoader ServiceR
 		return nil, fmt.Errorf("unable to validate service registration: please contact the auth service system admin to register your service - %v", err)
 	}
 
+	err = auth.dataLoader.GetAccessToken()
+	if err != nil {
+		return nil, fmt.Errorf("error getting access token: %v", err)
+	}
```

Comment:
I don't think we need to do this right away unless we are going to make a call which uses this token. Otherwise it is likely it will just expire before it is needed.